substrate/frame/election-provider-multi-block/src/signed/tests.rs

```diff
@@ -570,26 +611,8 @@ mod e2e {
 				]
 			);
 
-			assert_eq!(
-				verifier_events(),
-				vec![
-					crate::verifier::Event::Verified(2, 0),
-					crate::verifier::Event::Verified(1, 0),
-					crate::verifier::Event::Verified(0, 0),
-					crate::verifier::Event::VerificationFailed(0, FeasibilityError::InvalidScore),
-					crate::verifier::Event::Verified(2, 2),
-					crate::verifier::Event::Verified(1, 2),
-					crate::verifier::Event::Verified(0, 2),
-					crate::verifier::Event::Queued(
-						ElectionScore {
-							minimal_stake: 55,
-							sum_stake: 130,
-							sum_stake_squared: 8650
-						},
-						None
-					)
-				]
-			);
+			let remaining_events = verifier_events_since_last_call();
```

Comment:
The verifier pallet 100% won't do anything unless if it moves forward with `on_initialize()`, so I think the right thing
here would be to `roll_next()` at least once, but then ensure that nothing more happens. But maybe we already have
isolated tests for this further down.